schutzbot/deploy.sh

```diff
@@ -14,11 +19,19 @@ fi
 # Add osbuild team ssh keys.
 cat schutzbot/team_ssh_keys.txt | tee -a ~/.ssh/authorized_keys > /dev/null
 
-# Set up a dnf repository with the RPMs we want to test
+# Set up dnf repositories with the RPMs we want to test
 sudo tee /etc/yum.repos.d/osbuild.repo << EOF
 [osbuild]
 name=osbuild ${GIT_COMMIT}
-baseurl=http://osbuild-composer-repos.s3-website.us-east-2.amazonaws.com/osbuild/${ID}-${VERSION_ID}/${ARCH}/${GIT_COMMIT}
+baseurl=${DNF_REPO_BASEURL}/osbuild/${ID}-${VERSION_ID}/${ARCH}/${GIT_COMMIT}
+enabled=1
+gpgcheck=0
+# Default dnf repo priority is 99. Lower number means higher priority.
+priority=5
+
+[osbuild-composer]
```

Comment:
Until we make sure not to build `osbuild` as part of `osbuild-composer` CI, we must make sure to give the `osbuild` repo a higher priority (lower number). If we want to be consistent with my proposed `osbuild-composer` PR, we'd set `priority=10` on the `osbuild-composer` repository.

Reply:
Good point! Pushed a fix.